SecurityPkg/Include/Library/Tpm2DebugLib.h

```diff
@@ -40,4 +42,37 @@ DumpTpmOutputBlock (
   IN CONST UINT8  *OutputBlock
   );
 
+/**
+  This function dumps the provided event log.
+
+  @param[in]  EventLogFormat     The type of the event log for which the information is requested.
+  @param[in]  EventLogLocation   A pointer to the memory address of the event log.
+  @param[in]  EventLogLastEntry  If the Event Log contains more than one entry, this is a pointer to the
+                                 address of the start of the last entry in the event log in memory.
+  @param[in]  FinalEventsTable   A pointer to the memory address of the final event table.
+**/
+VOID
+EFIAPI
+DumpEventLog (
+  IN EFI_TCG2_EVENT_LOG_FORMAT    EventLogFormat,
+  IN EFI_PHYSICAL_ADDRESS         EventLogLocation,
+  IN EFI_PHYSICAL_ADDRESS         EventLogLastEntry,
+  IN EFI_TCG2_FINAL_EVENTS_TABLE  *FinalEventsTable
+  );
+
+/**
+  This function dumps the provided PCR digest.
+
+  @param[in]  PcrIndex   The index of the PCR.
+  @param[in]  HashAlg    The hash algorithm used.
+  @param[in]  Digest     The digest to be dumped.

```

Comment:
```suggestion
  @param[in]  PcrValues  The digest to be dumped.
```

Reply:
Thanks, I will go through and fix all of these. For the ones that were copied from the original code, I will do a follow on commit to address, so there is a commit that just moves the logic, then a commit that updates it. I think that is both easier to track what is going on and for upstreaming to edk2.